packages/docs/api/api-reference.md

```diff
@@ -152,6 +152,10 @@ The range is <expected - delta, expected + delta> (inclusive).
 
 Matches a iterable (array / set etc.) containing value
 
+#### expect.arrayWith(values: array of T)
```

Comment:
It's not array but rather multiple arguments, right?
so signature looks more like: `expect.arrayWith(...values: T)`

Reply:
Also, can you add it to the table of contents? can u add `containerWith` as well, I think I missed it...

Reply:
@krzkaczor well yeah, in the end it's a "varargs" :) I will add it to the table of contents.